spyder/plugins/switcher/widgets/switcheritem.py

```diff
@@ -0,0 +1,399 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's rename this file to simply be `item.py`

Reply:
Done :) [77a41d1](https://github.com/spyder-ide/spyder/pull/20714/commits/77a41d1e2033fd5d3f68242b459b35aa727df213)